packages/core/src/admin-ui/system/getAdminMetaSchema.ts

```diff
@@ -251,25 +259,25 @@ export function getAdminMetaSchema({
     },
   });
 
-  const KeystoneAdminUIListMeta = graphql.object<ListMetaRootVal>()({
-    name: 'KeystoneAdminUIListMeta',
+  const KeystoneAdminUIModelMeta = graphql.object<ModelMetaRootVal>()({
+    name: 'KeystoneAdminUIModelMeta',
     fields: {
       key: graphql.field({ type: graphql.nonNull(graphql.String) }),
       itemQueryName: graphql.field({
         type: graphql.nonNull(graphql.String),
       }),
-      listQueryName: graphql.field({
+      modelQueryName: graphql.field({
         type: graphql.nonNull(graphql.String),
       }),
```

Comment:
Since we're breaking all of this anyway, could you remove `itemQueryName` here (it is unused) and rename `modelQueryName` to like `graphqlPlural` or something?

Reply:
Sure!